common/cultures/rathnir_cultures.txt

```diff
@@ -366,4 +390,39 @@ luthic_group = {
 
 	}
 }
-	
\ No newline at end of file
+
+arcadian_group = {
+	graphical_culture = polynesiangfx
+
+	astanan = {
+		primary = A43 #Astana
+		
+		dynasty_names = {
+			Loulouna "Twin-Leaf" Jusepp
+		}
+	}
+	denthrosi = {
+		primary = A46 #Denthros
+
+		dynasty_names = {
+			Valhorn "Twin-Leaf"
+		}
+	}
+	kawalapuan = {
+		primary = A07 #Kawalapua
+
+		dynasty_names = {
+			Pantha Santha
+		}
+	}
+
+	dynasty_names = {
+		Valhorn "Twin-Leaf" Pantha Firnbolg Vishji

```

Comment:
repeated surnames; @Jabberdrake please clarify the EU4 syntax, will this cause issues?

Reply:
these should not cause issues, I dont know why, but they shouldn't